kolibri/plugins/coach/assets/src/views/plan/CreateExamPage/CreateQuizSection.vue

```diff
@@ -0,0 +1,183 @@
+<template>
+
+  <div>
+    <KGrid
+      class="add-padding"
+    >
+      <KGridItem
+        :layout4="{ span: 1 }"
+        :layout8="{ span: 1 }"
+        :layout12="{ span: 1 }"
+      >
+        <KIcon
+          icon="quiz"
+          class="style-icon"
+        />
+      </KGridItem>
+
+      <KGridItem
+        :layout4="{ span: 3 }"
+        :layout8="{ span: 7 }"
+        :layout12="{ span: 11 }"
+      >
+        <KTextbox
+          ref="title"
+          v-model.trim="examTitle"
+          :label="coachString('titleLabel')"
+          :autofocus="true"
+          :maxlength="100"
+        />
+      </KGridItem>
+    </KGrid>
+
+    <p>{{ $tr('addSectionsDescription') }}</p>
+
+    <hr class="bottom-border">
+    <br>
+
+    <KGrid style="margin:0;padding:0">
+      <KGridItem
+        :layout12="{ span: 8 }"
+      >
+        <span class="active-section">
+          {{ ($tr('sectionLabel')).toUpperCase() }}
+        </span>
+      </KGridItem>
```

Comment:
This ought to implement the [KTabs component ](https://develop--kolibri-design-system.netlify.app/ktabs/) which should give us an interface for handling the user's click action(s) of these headings.